requests/test/src-2/requests/Scala2RequestTests.scala

```diff
@@ -14,31 +14,33 @@ object Scala2RequestTests extends TestSuite{
             "https://httpbin.org/post",
             data = Map("hello" -> "world", "foo" -> "baz"),
             chunkedUpload = chunkedUpload
-          ).text()
+          ).text
           assert(read(res1).obj("form") == Obj("foo" -> "baz", "hello" -> "world"))
         }
       }
+
       test("put") {
         for (chunkedUpload <- Seq(true, false)) {
           val res1 = requests.put(
             "https://httpbin.org/put",
             data = Map("hello" -> "world", "foo" -> "baz"),
             chunkedUpload = chunkedUpload
-          ).text()
+          ).text
           assert(read(res1).obj("form") == Obj("foo" -> "baz", "hello" -> "world"))
         }
       }
-      test("send"){
-        requests.send("get")("https://httpbin.org/get?hello=world&foo=baz")
 
-        val res1 = requests.send("put")(
-          "https://httpbin.org/put",
-          data = Map("hello" -> "world", "foo" -> "baz"),
-          chunkedUpload = true
-        ).text
+    test("send"){
+            requests.send("get")("https://httpbin.org/get?hello=world&foo=baz")
```

Comment:
What happened with formatting here?

Reply:
The formatting complaints are more obvious in the editor. Yeah, I can twiddle them.